README.md

```diff
@@ -86,7 +86,7 @@
 - [terraform.py](https://github.com/mantl/terraform.py) - Ansible dynamic inventory script for parsing Terraform state files.
 - [ansible-navigator](https://github.com/ansible/ansible-navigator) - A text-based user interface (TUI) for Ansible.
 - [squest](https://hewlettpackard.github.io/squest/) - Self-service portal for Ansible Tower job templates.
-- [ansible-bender](https://ansible-community.github.io/ansible-bender/build/html/index.html) - Tool which bends containers using Ansible playbooks and turns them into container images.
```

Comment:
I am able to access the URL [https://ansible-community.github.io/ansible-bender/build/html/index.html](https://ansible-community.github.io/ansible-bender/build/html/index.html).

Could you try accessing it again after clearing the browser cache?